agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4216,6 +4219,38 @@ bool slave_thread::handle_client_association_request(Socket *sd, ieee1905_1::Cmd
     return send_cmdu_to_controller(cmdu_tx);
 }
 
+bool slave_thread::handle_1905_higher_layer_data_message(Socket *sd,
+                                                         ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    //only one backhaul manager (the slave) should return ACK for higher layer data message.
```

Comment:
```suggestion
    // Only one backhaul manager (the slave) should return ACK for higher layer data message.
```
or anything else you think can make it clearer.

In general - although it is porting existing code from point A to point B, you can always rework is where you find it suitable.
You can use there small PRs to ramp on the coding guidelines and enforce them to the ported code.